_sass/components/_actions.scss

```diff
@@ -9,12 +9,15 @@
 ul.actions {
   @include vendor("display", "flex");
   cursor: default;
+  flex-wrap: wrap;
   list-style: none;
-  margin-left: (_size(element-margin) * -0.5);
+  margin: (_size(element-margin) * -0.5) (_size(element-margin) * -0.5)
```

Comment:
Got the idea from [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Mastering_Wrapping_of_Flex_Items#Creating_gutters_between_items).

Reply:
Cool @ajyong! I've not seen this before. Can't wait for `row-gap` and `column-gap` like Grid Layout as they make it so much easier.

Reply:
Yeah CSS Grid is amazing, but flexbox still has its uses!  Just learned about this gutter-forming technique so skills++